src/Instrumentation/ReactHTTP/composer.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "open-telemetry/opentelemetry-auto-react-http",
+  "description": "OpenTelemetry auto-instrumentation for ReactPHP's HTTP Browser.",
+  "keywords": ["opentelemetry", "otel", "open-telemetry", "tracing", "reactphp", "async", "instrumentation"],
+  "type": "library",
+  "homepage": "https://opentelemetry.io/docs/php",
+  "readme": "./README.md",
+  "license": "Apache-2.0",
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "require": {
+    "php": "^8.1",
+    "ext-opentelemetry": "*",
+    "open-telemetry/api": "^1.0",
+    "open-telemetry/sem-conv": "^1.33",
```

Comment:
I think we've only released up to 1.32

Reply:
Derp. I fixed that in the example, forgot about composer. Will update.